hcipy/coronagraphy/vortex.py

```diff
@@ -373,9 +373,18 @@ def backward(self, instance_data, wavefront):
 
 		for i, (jones_matrix, prop) in enumerate(zip(instance_data.jones_matrices, instance_data.props)):
 			if i == 0:
-				pup = Wavefront(prop.backward(wavefront.electric_field))
+				if not wavefront.is_polarized:
+					efield = Wavefront(wavefront.electric_field, input_stokes_vector=(1, 0, 0, 0))
+					pup = Wavefront(prop.backward(efield.electric_field),
+									input_stokes_vector=efield.input_stokes_vector)
```

Comment:
I prefer to have the indentation be like this:
```python
pup = Wavefront(
    prop.backward(efield.electric_field),
    input_stokes_vector=efield.input_stokes_vector
)
```
if stuff doesn't fit on a single line. Another artifact from using tabs for indentation.